db/schema_0045-response-priorities.sql

```diff
@@ -0,0 +1,20 @@
+BEGIN;
+
+CREATE TABLE response_priorities (
+    id serial not null primary key,
+    body_id int references body(id) not null,
+    name text not null,
+    deleted boolean not null default 'f',
+    unique(body_id, name)
+);
+
+CREATE TABLE contact_response_priorities (
+    id serial NOT NULL PRIMARY KEY,
+    contact_id int REFERENCES contacts(id) NOT NULL,
+    response_priority_id int REFERENCES response_priorities(id) NOT NULL
+);
+
+ALTER TABLE problem
+    ADD COLUMN response_priority_id int REFERENCES response_priorities(id);
```

Comment:
I thought about it, and similar to `users.area_id` thought it's likely we'll want to index/filter on this field in the near future.